onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1132,6 +1150,70 @@ def _should_skip_test_sample(op_name: str, sample) -> Optional[str]:
     return None
 
 
+class OrtAbortedError(RuntimeError):
+    """ONNX Runtime Aborted."""
+
+
+def _ort_session_run(serialized_model, ort_inputs, return_dict) -> None:
+    """Run a model with ONNX Runtime and store the results in return_dict."""
+
+    # Handle SIGSEGV so Python does not abort
+    def sig_handler(signum, frame):
+        warnings.warn(f"Segmentation error ({signum}) detected in ORT:\n{frame}")
+
+    signal.signal(signal.SIGSEGV, sig_handler)
+
+    # Disable all ORT optimizations
+    session_options = onnxruntime.SessionOptions()
+    session_options.graph_optimization_level = (
+        onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
+    )
+    try:
+        session = ort.InferenceSession(serialized_model, session_options)
+        results = session.run(None, ort_inputs)
+        return_dict["results"] = results
+        return_dict["error"] = None
+    except Exception as e:  # pylint: disable=broad-except
+        return_dict["results"] = None
+        return_dict["error"] = e
+
+
+def _safe_ort_session_run(serialized_model: bytes, ort_inputs: Mapping[str, Any]):
+    """Run a model with ONNX Runtime in a separate process.
+
+    Args:
+        serialized_model: Serialized ONNX model proto.
+        ort_inputs: Inputs to the model.
+
+    Returns:
+        The inference result.
+
+    Raises:
+        OrtAbortedError if the process did not execute successfully.
+    """
+    manager = multiprocessing.Manager()
+    return_dict = manager.dict()
+    process = multiprocessing.Process(
```

Comment:
Do you have a measure of performance impact, if any?

Reply:
Looks like it's running 10x slower on windows and mac in some cases. I don't know if it was github actions or this change. Linux was fine

Reply:
No good. I will create a local option instead

Reply:
Looks like `spawn` is the only option for windows and mac :( https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods

I'm not sure if it is applicable to utilize process pool or some way to persist the process. If it's too much effort I guess local is good for now.

Reply:
I am still trying to figure out how to manage a pool when some of the workers can fail